lib/std/heap.zig

```diff
@@ -61,6 +61,23 @@ pub const max_page_size: usize = switch (builtin.os.tag) {
         .riscv32, .riscv64 => missing_max_page_size,
         else => missing_max_page_size,
     },
+    .netbsd => switch (builtin.cpu.arch) {
+        .x86, .x86_64 => 4 << 10,
+        // Alpha
+        .thumb, .thumbeb, .arm, .armeb, .aarch64, .aarch64_be => missing_max_page_size,
+        // https://github.com/NetBSD/src/blob/bd34fa0e7331e54a77f1d095e88c69f77ad222bd/sys/arch/mips/include/vmparam.h#L73
+        .mips, .mipsel, .mips64, .mips64el => 16 << 10,
+        // https://github.com/NetBSD/src/blob/bd34fa0e7331e54a77f1d095e88c69f77ad222bd/sys/arch/powerpc/include/vmparam.h#L41
+        .powerpc, .powerpc64, .powerpc64le, .powerpcle => 16 << 10,
+        .sh => missing_max_page_size,
+        // https://github.com/NetBSD/src/blob/bd34fa0e7331e54a77f1d095e88c69f77ad222bd/sys/arch/sparc/include/vmparam.h#L69
+        .sparc => 8 << 10,
+        .sparc64 => missing_max_page_size,
+        .riscv32, .riscv64 => missing_max_page_size,
+        // https://github.com/NetBSD/src/blob/bd34fa0e7331e54a77f1d095e88c69f77ad222bd/sys/arch/m68k/include/vmparam.h#L70
+        // m68k
```

Comment:
We do have an arch tag for m68k.

Reply:
Interesting that it's not even mentioned in Tier 4 Support.

Reply:
It's because it's an experimental target in LLVM and we don't have a self-hosted backend for it either. So if you want to play with it, you have to build Zig from source. Same for arc, csky, and hexagon.